cloudify/utils.py

```diff
@@ -211,7 +211,14 @@ def get_manager_rest_service_host():
     """
     Returns the host the manager REST service is running on.
     """
-    return os.environ[constants.REST_HOST_KEY].split(',')
+    host_ip = None
+    try:
+        # Context could be not available sometimes
+        host_ip = _get_current_context().host_ip
+    except RuntimeError:
+        # Fallback to read from environment variable
+        pass
+    return host_ip or os.environ[constants.REST_HOST_KEY].split(',')
```

Comment:
this function can be written simpler, just
```python
try:
   return _get_current_context().host_ip
except RuntimeError:
    return os.enviorn[..]
```

Reply:
@tehasdf You are right but what if the `_get_current_context().host_ip` was `None` for some reason ? this is why I implemented in this way to make sure if the `_get_current_context().host_ip` was `None` to fallback from `enviorn`

Reply:
good question what then :)
i'd say that it shouldn't really happen, with the way we have it written right now
maybe the fallback is ok after all